resources/js/functions.js

```diff
@@ -196,53 +196,37 @@ $(document).ready(function() {
   }
 });
 
-// Scala in the browser
 $(document).ready(function() {
-  if ($("#scastie-textarea").length) {
-    var editor =
-      CodeMirror.fromTextArea(
-        document.getElementById("scastie-textarea"),
-        {
-          // lineNumbers: false,
-          matchBrackets: true,
-          theme: "monokai",
-          mode: "text/x-scala",
-          autoRefresh: true,
-          fixedGutter: false,
-          extraKeys: {
-            'Ctrl-Enter': 'run',
-            'Cmd-Enter': 'run'
-          }
-        }
-      );
-
-    editor.setSize("100%", ($("#scastie-code-container").height()));
-
-    var codeSnippet = "List(\"Hello\", \"World\").mkString(\"\", \", \", \"!\")";
-    editor.getDoc().setValue(codeSnippet);
-    editor.refresh();
-
-    function run(){
-      console.log("run");
-      var scastieBaseUrl = "https://scastie.scala-lang.org";
-
-      $.ajax(
-        {
-          type: "POST",
-          url: scastieBaseUrl + '/scala-lang',
-          data: editor.getDoc().getValue(),
-          success: function(url) {
-            window.open(scastieBaseUrl + "/" + url);
-          },
-          // otherwise it's considered a popup
-          async: false
-        }
-      )
-    }
 
-    $('.btn-run').click(run);
-    CodeMirror.commands.run = run;
-  }
+  var snippets = [
+    "liEaE8NoSlqYhcmD1yvgUw",
+    "mcmef1f6TMCX8PQZm5X0Tw",
+    "R1txDbFnSwKzxVJ39ffgYw",
+    "yzZaG0TkTPasRnYvYNNLcw",
+    "64UAY3UrRNyeo1XVDV5j0Q",
+    "efJw6gtqQKu5AC75B0LNVA",
+    "CtQirsr6QjW0R4v3VkzJVA",
+    "BqYOO4d9SnSv8lVpNZgftw",
+    "CGHJ8X2vQQSJshHyplcPHA",
+    "NcTSnSx7T3qeoiTlEkn0cw",
+    "pGUX2rbRT96ZnSWYQxeUNg",
+    "3e166cHFR4qeQXuwSZh5EA",
+    "QU48GWJATH6cctgVwVcsmQ",
+    "sTbtRfGyQNSCSy108HiJzA",
+    "CM5gMYT4SkyPmWGX2whZyQ",
+    "FgnyudFsT7CRuFLz7wRD1Q",
+    "UBY0uNu6S0Kor5lcQSjrjg",
+    "bKjfsVlSSjC97bczClvNjA",
+    "ZIEdUV1ZT32sQe3aRP7Bug",
+    "AMXnCpvsSlq23834HJSrNg"
```

Comment:
I’m a bit scared of putting all these snippets on scala-lang. They are interesting pieces of code but a majority of them is a bit hard to understand. Also, you should credit the authors.